### ISSUE TITLE
Add slogan property analogous to logo

### ISSUE THREAD

**Issue by vholland** [schemaorg/schemaorg#2112]
We have https://schema.org/logo for specifying a specific image is a logo. These same sorts of organizations, brands, services, etc often have slogans. We should add ```slogan``` to the same domains as logo so authors have a specific text property for slogans rather than using description.

**Comment by thadguidry** [comment]
+1 slogan: "Just Do It."

**Comment by nicolastorzec** [comment]
Make sense. See drafted PR.

**Comment by MichaelAndrews-RM** [comment]
One different between logos and slogans.  Brands normally have one logo, but can have multiple slogans.  Slogans can be associated with brands, products, organizations, services, places.  Some slogans a trademarked, others are too generic to qualify for a trademark.

**Comment by vholland** [comment]
Schema.org has never had a policy on whether certain properties are unique or not. We have always left it to the data consumers to determine whether they wish to receive multiple values or not.

@nicolastorzec Thanks for the pull request!

**Comment by MichaelAndrews-RM** [comment]
@vholland I'm sure marketers will be interested in this property, so I hope that the schema.org documentation will provide a good definition and example.  For example, Coca Cola has had many slogans over the years such as "Coke is it!" and the current one ("Taste the Feeling").  It would be nice to show and example of multiple slogans in markup, so that it's clear to webmasters that more than one is allowed.  

I also suggest a definition for the slogan property that slogan is a phrase includes taglines and mottos (some marketers distinguish these things, but schema.org probably doesn't need to.)

**Comment by VladimirAlexiev** [comment]
+1 to list alias `motto`. Not sure about `tagline` because that means a different thing in the news industry.

> multiple slogans

Is there a way to indicate the "start date / end date" then?

**Comment by nicolastorzec** [comment]
The label in the PR is just: `Associated slogan`.

We could go with `An associated slogan or motto` or ` A slogan of motto associated with the item` to mimic the style of the other property labels. I'll update the PR once there is a consensus.

There is no straightforward way to associate a property like `slogan` or `logo` with a particular period or geography. Schema.org usually focuses on the here & now.

We would need to reify and/or use a construct similar to [role](http://blog.schema.org/2014/06/introducing-role.html) to do so, but I'm not sure it is worth it?

**Comment by VladimirAlexiev** [comment]
if @MichaelAndrews-RM is right that mottos change with some frequency, it's worth providing such example markup.

**Comment by nicolastorzec** [comment]
FYI: updated the comment to `A slogan of motto associated with the item.`

**Comment by MichaelAndrews-RM** [comment]
@VladimirAlexiev Here is an example of how many slogans can be associated with one company over time and in different countries.  McDonalds has numerous slogans they've used.  A comprehensive list is [available on Wikipedia ](https://en.wikipedia.org/wiki/List_of_McDonald%27s_ad_programs)of the slogans McDonalds uses in various countries.  In some markets these are translations, but sometimes markets using the same language use different slogans.

**Comment by RichardWallis** [comment]
Using the [Role](https://schema.org/Role) type, as suggested, would result in something similar to the following markup.
```
{
    "@context": "http://schema.org",
    "@type": "Organization",
    "name": "McDonalds",
    "slogan": [
        {
            "@type": "Role",
            "slogan": "Ronald Makes it Magic",
            "startDate": "1995-02-17",
            "endDate": "1997-10-15"
        },
        {
            "@type": "Role",
            "slogan": "Put a Smile On",
            "startDate": "2000",
            "endDate": "2003"
        },
        {
            "@type": "Role",
            "slogan": "I'm lovin' it",
            "startDate": "2003"
        }
    ]
}
```
For more detail a new [Role](https://schema.org/Role) subtype (_SloganCoverage_  ?) could be proposed which includes [areaserved](https://schema.org/areaServed) as a property.

**Comment by Abdulmanahtalib** [comment]
Support

**Comment by rvguha** [comment]
This is really not the kind of use originally imaged for Role.

We really should not have subclasses of Role for different properties.

guha

**Comment by VladimirAlexiev** [comment]
I really like the McDonald slogan examples, but @RichardWallis I think `SloganCoverage` is too specific. I can give 2 more examples of a similar phenomenon:
- the same **product**  is very often called differently in different countries. Eg a drug called `Omnic Tocas` in BG is called `Omnic Ocas` in ES. This is done for marketing or cultural sensitivity reasons.
- CIDOC CRM has an activity class  http://personal.sirma.bg/vladimir/crm/entity_list_cleaned.html#E15_Identifier_Assignment that allows to describe when and by whom a certain **person or place** name has been used

@rvguha what is your objection about `Role`: that it shouldn't be used to describe the temporal/spatial coverage of a slogan (or other name), or that subclasses of `Role` should not be created?

Looking at https://schema.org/Role, I think the following props should be added to it: `location`, `areaServed` and maybe `spatialCoverage` (there's a large overlap between the last two).

**Comment by RichardWallis** [comment]
Implemented in [release 3.5 ](https://schema.org/docs/releases.html#v3.5)
